festim/subdomain/surface_subdomain.py

```diff
@@ -0,0 +1,42 @@
+from dolfinx import fem
+import numpy as np
+
+
+class SurfaceSubdomain1D:
+    """
+    Surface subdomain class for 1D cases
+
+    Args:
+        id (int): the id of the surface subdomain
+        x (float): the x coordinate of the surface subdomain
+
+    Attributes:
+        id (int): the id of the surface subdomain
+        x (float): the x coordinate of the surface subdomain
+        dofs (np.array): the dofs of the surface subdomain
+
+    Usage:
+        >>> surf_subdomain = F.SurfaceSubdomain1D(id=1, x=1)
+    """
+
+    def __init__(self, id, x) -> None:
+        self.id = id
+        self.x = x
+
+        self.dofs = None
```

Comment:
Looking at this now, I don't think we use SurfaceDomains.dofs anywhere in the code do we?